keel-core/src/main/kotlin/com/netflix/spinnaker/keel/persistence/CombinedRepository.kt

```diff
@@ -358,8 +358,8 @@ class CombinedRepository(
   override fun getArtifact(name: String, type: ArtifactType, reference: String, deliveryConfigName: String): DeliveryArtifact =
     artifactRepository.get(name, type, reference, deliveryConfigName)
 
-  override fun getArtifact(deliveryConfigName: String, reference: String, type: ArtifactType): DeliveryArtifact =
```

Comment:
just for my understanding, what used to be the values for `type`?

Reply:
`deb` or `docker`. It used to be that artifacts didn't have a reference, so you could refer to them by name + type. But now, we have a reference, and we validate that it's unique in your delivery config. So we need to use that everywhere